scripts/mkcloud

```diff
@@ -1251,14 +1253,24 @@ function sanity_checks()
 
     # always fetch the latest version
     if [ -z "$NOQACROWBARDOWNLOAD" ] ; then
-        #wget -O qa_crowbarsetup.sh "https://raw.githubusercontent.com/SUSE-Cloud/automation/master/scripts/qa_crowbarsetup.sh"
-        # Switch to curl as wget has an issue with github: https://github.com/netz98/n98-magerun/issues/75
-        curl -s -o qa_crowbarsetup.sh "https://raw.githubusercontent.com/SUSE-Cloud/automation/master/scripts/qa_crowbarsetup.sh"
+        if [ ! -e $(dirname $me)/$qa_crowbarsetup ] ; then
```

Comment:
we could add a variable  git_qa_crowbarsetup=$(dirname $me)/$qa_crowbarsetup to reduce duplication and make the code easier to understand